code/datums/spells/cluwne.dm

```diff
@@ -0,0 +1,86 @@
+/obj/effect/proc_holder/spell/targeted/touch/cluwne
+	name = "Curse of the Cluwne"
+	desc = "Turns the target into a fat and cursed monstrosity of a clown."
+	hand_path = /obj/item/weapon/melee/touch_attack/cluwne
+
+	school = "transmutation"
+
+	charge_max = 600
+	clothes_req = 1
+	cooldown_min = 200 //100 deciseconds reduction per rank
+
+	action_icon_state = "clown"
+
+/mob/living/carbon/human/proc/makeCluwne()
+	to_chat(src, "<span class='danger'>You feel funny.</span>")
+	adjustBrainLoss(80)
+	nutrition = 9000
+	confused = 120
+	if(mind)
+		mind.assigned_role = "Cluwne"
+
+	var/obj/item/organ/internal/honktumor/cursed/tumor = new
+	tumor.insert(src)
+	mutations.Add(NERVOUS)
+	dna.SetSEState(NERVOUSBLOCK, 1, 1)
+	genemutcheck(src, NERVOUSBLOCK, null, MUTCHK_FORCED)
+
+	animate_clownspell(src)
+
+	unEquip(w_uniform, 1)
+	unEquip(shoes, 1)
+	unEquip(gloves, 1)
+	if(!istype(wear_mask, /obj/item/clothing/mask/cursedclown)) //Infinite loops otherwise
+		unEquip(wear_mask, 1)
+	equip_to_slot_if_possible(new /obj/item/clothing/under/cursedclown, slot_w_uniform, 1, 1, 1)
+	equip_to_slot_if_possible(new /obj/item/clothing/gloves/cursedclown, slot_gloves, 1, 1, 1)
+	equip_to_slot_if_possible(new /obj/item/clothing/mask/cursedclown, slot_wear_mask, 1, 1, 1)
+	equip_to_slot_if_possible(new /obj/item/clothing/shoes/cursedclown, slot_shoes, 1, 1, 1)
+	real_name = "cluwne"
+
+/mob/living/carbon/human/proc/makeAntiCluwne()
+	to_chat(src, "<span class='danger'>You don't feel very funny.</span>")
+	adjustBrainLoss(-120)
+	nutrition = NUTRITION_LEVEL_STARVING
```

Comment:
You'll also want to possibly reset `overeatduration` here if you want to make people thin